src/core/annotation.js

```diff
@@ -4811,11 +4811,33 @@ class StampAnnotation extends MarkupAnnotation {
     this.data.noHTML = false;
   }
 
+  static encodePNGFilter(data, columns) {
+    if (data.length % columns !== 0) {
+      return null;
+    }
+    let encoded_data = [];
+
+    let colorCount = 3;
+    let encoded_count = 0;
```

Comment:
Please note that as outlined in https://github.com/mozilla/pdf.js/wiki/Style-Guide#naming variable names should *not* contain underscores.
```suggestion
    let encodedData = [];
    let colorCount = 3;
    let encodedCount = 0;
```

---

Also, it seems like there's some lint failures in this patch; please remember to run `gulp lint` locally during development.